cmsplugin_articles_ai/factories.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+import random
+import string
+from datetime import timedelta
+
+import factory
```

Comment:
I think the factories are good thing to keep in the package so that users can take advantage of the built int factories. Not sure about tests though.. those are probably just excess weight and can be left out from the installable package.